tools/prune_database.sh

```diff
@@ -49,4 +49,6 @@ fi
 
 # Remove media files (because they are no longer usable without the corresponding database entries)
 rm -rfv "${PACKAGE_DIR:?}/media"
+mkdir "${PACKAGE_DIR:?}/media"
+cp "${PACKAGE_DIR:?}/static/images/fallback_title_image.png" "${PACKAGE_DIR?:}/media/fallback_title_image.png"
```

Comment:
Does this make sense? I think we should rather adapt the fallback path to reference the image in `/static/` than to move an image into the media directory which is not handled by the normal Django upload storage. This will also cause overhead on all production systems if we have to move this file manually.

Reply:
Yes, this is a really ugly workaround.
Unfortunately I can not get the fallback image to be looked for anywhere but the media dir. I re-did the changes I originally thought would work, could you have another look?

(Thanks for all the reviews btw!)

Reply:
Ah ok I get it, sorry I wasn't aware that the default value is stored on the image field.
Hmm, what do you think about creating a new property on the offer model and leave the image field in the db empty per default? Something like this:
```
def title_image_url(self):
    return self.title_image.url if self.title_image else settings.DEFAULT_TITLE_IMAGE
```
This would require to replace `title_image.url` with `title_image_url` in the templates and to remove the custom handling in the forms, so not sure whether it's worth it.
I would also be fine with the previous solution of copying the file manually...

Reply:
I like the idea, but would prefer keeping the handling in the forms; I'll try to figure out a way to do both.

Reply:
Alright, I think I found an acceptable workaround. The field is now nullable, and the templates just have to check if the images exist, otherwise display the static fallback image.